src/util/c_types.cpp

```diff
@@ -111,6 +111,11 @@ typet uint_type()
   return unsignedbv_typet(config.ansi_c.int_width);
 }
 
+typet uint_type(unsigned int size)
```

Comment:
This is not needed. Check how we build _ExtInt in the C frontend.

Reply:
The issue here is that `c_types` regards... C base types. Here you are basically creating an alias. Which is not needed.

Reply:
This makes sense, I have a redundant alias and as this is for solidity there is no reason for dependence on c_types. Will amend to use the unsignedbv_typet.